src/components/storage/CockpitStorageIntegration.jsx

```diff
@@ -130,42 +138,90 @@ export const CockpitStorageIntegration = ({
         }
     }, [onCritFail]);
 
+    const handleConfirmOpenModal = () => {
+        setShowConfirmation(false);
+        setConfirmed(true);
+        setShowStorage(true);
+    };
+
+    const handleCancelOpenModal = () => {
+        setShowConfirmation(false);
+        setShowStorage(false);
+        setConfirmed(false);
+    };
+
     return (
-        <Modal
-          aria-label={_("Configure storage")}
-          className={backdropClass + " " + idPrefix + "-modal-page-section"}
-          footer={<ReturnToInstallationButton onAction={() => setShowDialog(true)} />}
-          hasNoBodyWrapper
-          isOpen={showStorage}
-          onClose={() => setShowDialog(true)}
-          showClose={false}
-          variant={ModalVariant.large}>
-            <Alert
-              isInline
-              title={_("Changes made here will immediately affect the system. There is no 'undo'.")}
-              variant="warning"
-            />
-            <Divider />
-            <div className={idPrefix + "-page-section-cockpit-storage"}>
-                <PageSection>
-                    <iframe
-                      src="/cockpit/@localhost/storage/index.html"
-                      name="cockpit-storage"
-                      id="cockpit-storage-frame"
-                      className={idPrefix + "-iframe-cockpit-storage"} />
-                </PageSection>
-                <ModifyStorageSideBar />
-            </div>
-            {showDialog &&
-            <CheckStorageDialog
-              dispatch={dispatch}
-              isFormDisabled={isFormDisabled}
-              onCritFail={onCritFail}
-              scenarioAvailability={scenarioAvailability}
-              setShowDialog={setShowDialog}
-              setShowStorage={setShowStorage}
-            />}
-        </Modal>
+        <>
+            <Modal
+              title="Modify storage"
+              titleIconVariant="warning"
+              aria-describedby="modal-title-icon-description"
+              isOpen={showConfirmation}
+              onClose={handleCancelOpenModal}
+              variant="small"
+              actions={[
+                  <Button
+                    id={idPrefix + "-enter-storage-confirm"}
+                    key="confirm"
+                    variant="warning"
+                    onClick={handleConfirmOpenModal}>
+                      {_("Launch storage editor")}
+                  </Button>,
+                  <Button
+                    id={idPrefix + "-enter-storage-cancel"}
+                    key="cancel"
+                    variant="link"
+                    onClick={handleCancelOpenModal}>
+                      {_("Cancel")}
+                  </Button>
+              ]}
+            >
+                <span id="modal-title-icon-description">
```

Comment:
You can make this span wrapper a TextContent.

Then you can use strong text and such stylings by using Text variants.  https://www.patternfly.org/components/text

Then you wont need the <br> i think